src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -65,6 +66,11 @@ public class Date {
                                                    DateTimeFormatterBuilder::appendOptional,
                                                    (builder, formatterBuilder) -> builder.append(formatterBuilder.toFormatter()))
                                            .toFormatter(Locale.US);
+
+        DATE_REGEX = "\\d{4}-\\d{1,2}-\\d{1,2}" + // covers YYYY-MM-DD, YYYY-M-DD, YYYY-MM-D, YYYY-M-D
```

Comment:
Oh, the comment referencing `org.jabref.model.entry.Date#parse(java.lang.String)` is gone.

```suggestion
        // There is also {@link org.jabref.model.entry.Date#parse(java.lang.String)}.
        // The regex of that method cannot be used as we parse single dates here and that method parses 
        //   i) date ranges and ii) two dates separated by '/'
        DATE_REGEX = "\\d{4}-\\d{1,2}-\\d{1,2}" + // covers YYYY-MM-DD, YYYY-M-DD, YYYY-MM-D, YYYY-M-D
```